src/amo/css/inc/vars.scss

```diff
@@ -23,14 +23,13 @@ $default-font-family: 'Clear Sans', Helvetica, Arial, sans-serif;
 
 // Font-Size Variables
 $default-line-height: 19px;
```

Comment:
might cause unintended consequences but generally we'd increase the default line-height if the default font size increased?

Reply:
That reminds me that the style guide has specific line heights for each size. I'll make sure this one is decent, and then maybe we should have a follow-up to update them all. It's not as straightforward as this patch as I don't think we have standard fonts defined with multiple properties, but we probably should.

Reply:
I changed this from `19px` to `24px`, which is what is suggested for Body MD in the design guide. I am going to file two follow-ups as well, one for reducing the number of font sizes and one for using suggested line heights along with the new font sizes.